ParisBNF/abb/BNFabb134.xml

```diff
@@ -0,0 +1,452 @@
+                        </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="140">140</measure>
+                                    <dimensions type="outer" unit="mm">
+                                      <height>128</height>
+                                      <width>120</width>
+                                    </dimensions>
+                                </extent>
+
+                                <foliation>Foliation in the upper right corner of the folia.
+                                  </foliation>
+                                  <collation>The quire structure cannot be discerned due to the tight binding.
+                                    </collation>
+
+
+
+                                  <condition key="deficient">
+                                      The quires have been dismantled. The text block is now composed of single folia, each
+                                      mounted on a paper guard. A few leaves have been misplaced.</condition>
+
+                                </supportDesc>
+
+                           <layoutDesc>
+                              <layout columns="2" writtenLines="17">
+                                  <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is partly visible.</ab>
+                              <ab type="ruling" subtype="pattern" cert="medium">1A-1A-1A1A/0-0/0-0/J</ab>
+                              <ab type="ruling">The upper line is written below the ruling.</ab>
+                              <ab type="ruling">The bottom line is written above the ruling.</ab>
+                             <note>The leaves <locus target="#1 #139 #140"/> are neither pricked nor ruled.</note>
+                            </layout>
+                            <layout>
+                            <ab type="CruxAnsata">
+                               <locus from="2r" to="138v"/>The beginning of each text
+                                is marked with a crux ansata-sign with coronis.</ab>
+                              </layout>
+                           </layoutDesc>
+
+                          </objectDesc>
+                        <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                              <seg type="script">Late 13th/mid-14th century.</seg>
+                              <seg type="ink">Black; red.</seg>
+                              <desc>Careful, slightly irregular; by a good scribe.
+                                In their catalogues,
+                                Conti Rossini and Chaîne dated the manuscript to
+                                the <date>15th</date> and <date>14th/15th century</date>, resp., on the palaeographical grounds.
+                                Uhlig suggested the time around the middle or shortly before the end of the 14th century
```

Comment:
Could you include the bibliographic reference for Uhlig? (For us, it is clear, but we want to be transparent for everyone)

Reply:
Ok